guess-react/src/App.js

```diff
@@ -0,0 +1,97 @@
+import { useState } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
```

Comment:
Looks beautiful .... but for a newbit like me, I NEED comments so I know what I am looking at ... what do you think?